entity/contents/filter_contents.h

```diff
@@ -91,4 +106,37 @@ class BlendFilterContents : public FilterContents {
   FML_DISALLOW_COPY_AND_ASSIGN(BlendFilterContents);
 };
 
+/*******************************************************************************
+ ******* GaussianBlur1DFilterContents
+ ******************************************************************************/
+
+class GaussianBlur1DFilterContents final : public FilterContents {
```

Comment:
Perhaps call this `DirectionalGaussionBlurFilterContents` instead?

Reply:
Done.